Symfony/CS/ConfigurationException/InvalidConfigurationException.php

```diff
@@ -24,9 +24,10 @@ class InvalidConfigurationException extends \InvalidArgumentException
 {
     /**
      * @param string $message
+     * @param int    $code
      */
-    public function __construct($message)
+    public function __construct($message, $code = FixCommand::EXIT_STATUS_FLAG_HAS_INVALID_CONFIG)
```

Comment:
Maybe make the the default `null`, then do a ternary thing inside the code. That would be cleaner to me.